dist/tools/doccheck/exclude_patterns

```diff
@@ -14948,3 +14948,24 @@ boards/nucleo\-f439zi/include/periph_conf\.h:[0-9]+: warning: Member ETH_DMA_ISR
+pkg/nimble/autoadv/include/nimble_autoadv_params\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_TX_POWER \(macro definition\) of file nimble_autoadv_params\.h is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv_params\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_CHANNEL_MAP \(macro definition\) of file nimble_autoadv_params\.h is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv_params\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_OWN_ADDR_TYPE \(macro definition\) of file nimble_autoadv_params\.h is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv_params\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FILTER_POLICY \(macro definition\) of file nimble_autoadv_params\.h is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv_params\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_PARAMS \(macro definition\) of file nimble_autoadv_params\.h is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_LEGACY \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_HD_MODE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_CONNECTABLE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_SCANNABLE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_ANONYMOUS \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_SCAN_REQ_NOTIF \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_LEGACY \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_HD_MODE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_CONNECTABLE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_SCANNABLE \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_ANONYMOUS \(macro definition\) of group pkg_nimble_autoadv is not documented\.
+pkg/nimble/autoadv/include/nimble_autoadv\.h:[0-9]+: warning: Member NIMBLE_AUTOADV_FLAG_SCAN_REQ_NOTIF \(macro definition\) of group pkg_nimble_autoadv is not documented\.
```

Comment:
These should not be necessary anymore.

Reply:
Still needed, new Doxygen complains even if they are grouped

Reply:
Is this also the case when using `/**< comment. */` instead of `/* comment */` ?

Reply:
Thanks that worked!